site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -165,18 +165,41 @@ We need a firewall to allow UDP traffic to nodes tagged as `game-server` via por
 next section.
 
 ```bash
-gcloud compute firewall-rules create game-server-firewall \
+gcloud compute firewall-rules create gke-agones-game-server-firewall \
```

Comment:
🤔  why this name change?

Reply:
This is bleeding our internals a bit but it's a simple change that avoids our internal firewall enforcer.